portability-types-common/src/main/java/org/datatransferproject/types/common/models/music/MusicPlaylistItem.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2022 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.datatransferproject.types.common.models.music;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Preconditions;
+import java.util.Objects;
+
+/**
+ * POJO for MusicPlaylistItem
+ */
+public class MusicPlaylistItem {
```

Comment:
since this class doesn't have a schema.org parallel to lean on, could you make sure any primitive values are well described in javadoc? eg: what kinds of strings should and can be expected in `playlistId`? What does `order` mean? Is it required, what's it's intended use-case, etc.

Happy to help draft these! (though I think we have some initial drafts we can use in a doc somehere...)

Reply:
Added the javadoc in latest commit https://github.com/google/data-transfer-project/pull/1131/commits/0ada90d3d2f133187ea2df70db866b5beb6ffffb.